### PR TITLE
(feat) add simple commands for Output Monitor 1, Monitor 2, Monitor Auto

### PR DESCRIPTION
**Pull request by splattner**
contributes to #37

### CODE REVIEW

File: intg-denonavr/media_player.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import avr
 from config import AvrDevice, create_entity_id
 from ucapi import EntityTypes, MediaPlayer, StatusCodes
-from ucapi.media_player import Attributes, Commands, DeviceClasses, Features, States
+from ucapi.media_player import Attributes, Commands, DeviceClasses, Features, States, Options
```
Comment (zehnm) [inline intg-denonavr/media_player.py:14]: isort lint error: Imports are incorrectly sorted and/or formatted.
Fix with `python -m isort intg-denonavr/.`

File: intg-denonavr/media_player.py [hunk 2]
```diff
@@ -70,12 +70,23 @@ def __init__(self, device: AvrDevice, receiver: avr.DenonDevice):
...
+        self.simple_commands = [
+            "OUTPUT_1",
+            "OUTPUT_2",
+            "OUTPUT_AUTO"
+        ]
+
+        options = {
+            Options.SIMPLE_COMMANDS: self.simple_commands
+        }
+
```
Comment (zehnm) [inline intg-denonavr/media_player.py:82]: black linter error.
Fix with `python -m black intg-denonavr --line-length 120`

**Review by zehnm — CHANGES_REQUESTED** [review]
Great, thank you! I need to find an HDMI cable to test my unused HDMI out 2 port :-)

Please fix the linting errors, then I'll merge it and include it in the next release, scheduled for tomorrow.

You can leave the pylint `intg-denonavr/media_player.py:93:4: R0915: Too many statements (51/50) (too-many-statements)` issue, or add an ignore tag if you like.
The existing if-then-else monster just got too big. I'll replace it with the string case statement.

**Comment by splattner** [comment]
L'll check those. My python dev setup was not really ready/available so I did not lint or anything else... :see_no_evil:  sry, but I'll fix asap ;)

File: intg-denonavr/media_player.py (hunk 1 above)
Comment (splattner) [reply intg-denonavr/media_player.py]: fixed

File: intg-denonavr/media_player.py (hunk 2 above)
Comment (splattner) [reply intg-denonavr/media_player.py]: fixed

**Comment by splattner** [comment]
all fixed and I added a pylint disable for R0915. But I agree, this part should probably be solved a bit nicer

**Review by zehnm — APPROVED** [review]
Thanks again, looking good!

**Pull request merged**